src/notification/actionable-notification.component.ts

```diff
@@ -82,7 +82,9 @@ export class ActionableNotification extends BaseNotification {
 		this._notificationObj = Object.assign({}, this.defaultNotificationObj, obj);
 	}
 
-	@HostBinding("attr.id") notificationID = `notification-${ActionableNotification.notificationCount++}`;
+	@HostBinding("attr.id") notificationID = `notification-${ActionableNotification.notificationCount}`;
+	@HostBinding("attr.aria-label") notificationLabel = `Notification`;
+	@HostBinding("attr.aria-describedby") notificationDescription = `Notification-${ActionableNotification.notificationCount++}`;
```

Comment:
```suggestion
	notificationID = `notification-${ActionableNotification.notificationCount++}`;
	@HostBinding("attr.aria-labelledby") labelledby= `Notification-${this.notificationID}`; // May need to use a getter, double check.
```

Update the code to use `aria-labelledBy`, you can remove `aria-label`. The ID should be assigned to the following element (title):

```
<div *ngIf="!notificationObj.template" cdsActionableTitle [innerHTML]="notificationObj.title"></div>
```